cli/tests/test_cli.py

```
@@ -519,6 +531,36 @@ def test_request_invalid_json(self, mock_request):
+    @patch("cli.API_KEY", "test-key")
+    def test_request_validation_error_with_raise_flag(self, mock_request):
+        """Test that validation errors are raised when raise_on_validation_error=True."""
+        mock_response = Mock()
+        mock_response.status_code = 200
+        mock_response.raise_for_status.return_value = None
+        mock_response.json.return_value = {
+            "invalid": "data"
+        }  # This will fail validation
+        mock_request.return_value = mock_response
+
+        with pytest.raises(Exception):  # Should raise validation error
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)